jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/definition/list/ProcessDefinitionListPresenter.java

```diff
@@ -189,17 +197,11 @@ boolean onRuntimeDataServiceError(final Throwable throwable) {
     }
 
     @WorkbenchMenu
-    public void buildMenu(final Consumer<Menus> menusConsumer) {
-        menusConsumer.accept(MenuFactory
-                                     .newTopLevelCustomMenu(new RefreshMenuBuilder(this))
-                                     .endMenu()
-                                     .build());
-    }
-
-    protected void selectProcessDefinition(final ProcessSummary processSummary) {
-        setupDetailBreadcrumb(constants.ProcessDefinitionBreadcrumb(processSummary.getName()));
-        placeManager.goTo(PerspectiveIds.PROCESS_DEFINITION_DETAILS_SCREEN);
-        fireProcessDefSelectionEvent(processSummary);
+    public Menus buildMenu() {
```

Comment:
@bxf12315 other thing I noticed is the refresh button menu is not displayed anymore